utils/scripts/install-ubuntu.sh

```diff
@@ -21,7 +21,6 @@ apt-get install \
     clang-format \
     cmake \
     curl \
-    fuse3 \
```

Comment:
This might still be needed to get the suid fuse helper.  I can run some checks for this if you want.

Reply:
Honestly, not sure why I removed this one

Reply:
let's fix this in a follow on PR please @jolivier23

Reply:
This looks OK to me.  On both Rocky and ubuntu this PR removed the libfuse headers from /usr/include but kept both the library and fusermount binary so I don't think we need to make any more changes in this regard.